hera_sim/tests/test_vis.py

```diff
@@ -94,6 +94,111 @@ def test_values(self):
             v[:, 0, 1], 1 + np.exp(-2j * np.pi * np.sqrt(0.5)), 7
         )
 
+class TestSimRedData(unittest.TestCase):
```

Comment:
Other than knowing that this is testing `sim_red_data`, I have no idea what you are testing here. Could you supply a few higher-level comments?

Reply:
Done.